.circleci/config.yml

```
@@ -15,7 +15,7 @@ parameters:
 jobs:
     host_docs:
         machine:
-            image: ubuntu-2004:202111-01
+            image: ubuntu-2004:current
```

```suggestion
            image: ubuntu-2204:current
```

shouldn't it be more something like this?

https://circleci.com/developer/machine/image/ubuntu-2204

---

Oops yes that should be it